### PR TITLE
Update default nullable optional parameters sign

### PR DESCRIPTION
**Pull request by Kaian**
According to documentation, the format ClassName $paremeter = null will be deprecated in 8.4.0

More info at https://www.php.net/manual/en/functions.arguments.php

### CODE REVIEW

File: Entity/Serializer/Normalizer/EntityDenormalizer.php [hunk 1]
```diff
@@ -146,7 +146,7 @@ protected function denormalizeDateTimes($data, $class)
         return $response;
     }
 
-    private function denormalizeEntity(array $input, string $class, EntityInterface $entity = null, string $normalizationContext)
+    private function denormalizeEntity(array $input, string $class, ?EntityInterface $entity = null, string $normalizationContext)
```
Comment (danigargar) [inline Entity/Serializer/Normalizer/EntityDenormalizer.php]:
```suggestion
    private function denormalizeEntity(array $input, string $class, ?EntityInterface $entity, string $normalizationContext)
```

File: Entity/Serializer/Normalizer/EntityDenormalizer.php [hunk 2]
```diff
@@ -274,7 +274,7 @@ private function injectEqualValuesIfNotPresent(array $input, $criteria): array
      * @param DataTransferObjectInterface $dto
      * @return EntityInterface
      */
-    private function mapToEntity(string $class, EntityInterface $entity = null, DataTransferObjectInterface $dto): EntityInterface
+    private function mapToEntity(string $class, ?EntityInterface $entity = null, DataTransferObjectInterface $dto): EntityInterface
```
Comment (danigargar) [inline Entity/Serializer/Normalizer/EntityDenormalizer.php]:
```suggestion
    private function mapToEntity(string $class, ?EntityInterface $entity, DataTransferObjectInterface $dto): EntityInterface
```

**Review by danigargar — CHANGES_REQUESTED** [review]

File: Entity/Serializer/Normalizer/EntityDenormalizer.php (hunk 1 above)
Comment (Kaian) [reply Entity/Serializer/Normalizer/EntityDenormalizer.php]: Epic good catch!

**Comment by Kaian** [comment]
Updated, mark conversations as resolved if those are now fixed

**Review by danigargar — APPROVED** [review]

**Pull request merged**